lib/std/mem.zig

```diff
@@ -3522,10 +3522,14 @@ pub fn minMax(comptime T: type, slice: []const T) struct { min: T, max: T } {
     return .{ .min = minVal, .max = maxVal };
 }
 
+pub fn MinMaxResult(comptime T: type) type {
+    return struct { min: T, max: T };
+}
```

Comment:
This type + `IndexOfMinMaxResult` were required to be added after the change to make it possible to instantiate them. You couldn't use `const Result = @TypeOf(minMax(u8, ""))` because each invocation of the old `minMax` returned a new and distinct struct.